moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_param_widget.cpp

```diff
@@ -51,6 +51,8 @@ MotionPlanningParamWidget::MotionPlanningParamWidget(QWidget* parent) : rviz::Pr
 
 MotionPlanningParamWidget::~MotionPlanningParamWidget()
 {
+  if (property_tree_model_)
+    delete property_tree_model_;
```

Comment:
Same as above, if the MotionPlanningParamWidget is copyable or movable, this could lead to double frees.